application-form/src/components/share/multiForm/components/share/NavBar.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <div class="nav-container">
+    <div class="header" v-if="!isAdminPage">{{ header }}</div>
+    <div class="nav-bar">
+      <div
+        class="nav-item"
+        v-for="item in data"
+        :key="item.id"
+        @click="onChangeStep(item.step)"
+      >
+        <div
+          class="item-number"
+          :class="{
+            active: item.step === step,
+            activePrevItem: item.step <= step,
+          }"
+        >
+          <p>{{ item.step }}</p>
+        </div>
+        <p class="item-info">{{ item.title }}</p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    header: {
+      type: String,
+      default: () => "Register",
+    },
+    data: {
+      type: Array,
+      default: () => [],
+    },
+    step: {
+      type: Number,
+      default: () => 1,
+    },
+  },
+  methods: {
+    onChangeStep(choseStep) {
+      this.$emit("onChangeStep", choseStep);
+    },
+  },
+  computed: {
+    isAdminPage() {
+      let isAdmin = this.$route.name.includes("admin") ? true : false;
```

Comment:
let isAdmin = !!this.$route.name.includes("admin")
!! convert các giá trị truthy falsy thành boolean